test/parallel/test-dgram-send-callback-recursive.js

```diff
@@ -15,7 +15,8 @@ function onsend() {
   if (sent++ < limit) {
     client.send(chunk, 0, chunk.length, port, common.localhostIPv4, onsend);
   } else {
-    assert.strictEqual(async, true, 'Send should be asynchronous.');
+    assert.strictEqual(async, true, `Send should be asynchronous.
+      Saw ${sent} messages, expected fewer than ${limit}.`);
```

Comment:
Please do not use multiline template strings. This adds lots of whitespace to the message. Instead a string concat should be used here (and the template string in the second line).

Reply:
You might also just remove the message all together as that would also increase the readability a lot.

Reply:
@shakeelmohamed ... thank you so much for the pull request and for participating in the code and learn! As @BridgeAR indicates, multiline template strings are less than ideal and are not something that we use generally. Also, the added message does not quite fit the check. As @BridgeAR suggests, I would just remove the message altogether and leave it as `assert.strictEqual(async, true)`.

Reply:
Thanks for the feedback @BridgeAR @jasnell, I did the lazy person's force push (delete remote branch, squash locally, re-push branch) and had to create a new PR: #16029